### ISSUE TITLE
Invalid deb package on Ubuntu 19.10

### ISSUE THREAD

**Issue by carroarmato0** [keybase/keybase-issues#3617]
Looks like the same issue as #2104

```
$ sudo apt install ./keybase_amd64.deb 
Reading package lists... Error!
E: Problem parsing dependency 21 of keybase:amd64=4.7.0-20191021162721.67d9526320
E: Error occurred while processing keybase (NewVersion2)
E: Problem with MergeList /home/carroarmato0/keybase_amd64.deb
E: The package lists or status file could not be parsed or opened.

```

**Comment by carroarmato0** [comment]
I worked around the problem by adding the repository and gpg key manually.

```
# /etc/apt/sources.list
deb https://prerelease.keybase.io/deb stable main
# deb-src https://prerelease.keybase.io/deb stable main

Add the key from  https://keybase.io/docs/server_security/code_signing_key.asc
```

**Comment by maxtaco** [comment]
Thanks for the report, we'll look into it today

cc @heronhaye

**Comment by carroarmato0** [comment]
For completeness, after the installation one should remove the entry from /etc/apt/sources.list as the keybase package will have added its own entry in sources.list.d

**Comment by remedycoin** [comment]
I can confirm also having had the problem and the proposed workaround worked for me.  

Once I downloaded the .asc key file I used this to add it to the apt keyring:
sudo apt-key add ./code_signing_key.asc

Thanks for the great work on Keybase!

I'd like to talk with someone about adding ring of trust features to the keybase project.  I'm working on RemedyCoin where I hope to have users express various parameters of trust about one another in regards to creating financial instruments where forgiveness is the established underlying value.  

Any ideas on who to talk to about this?
